File: python/10783.py
for i in range (int(input())):
	a, b = int(input()), int(input())
	if a % 2 == 0:
		a += 1
	if b % 2 == 0:
		a -= 1
	print(f"Case {i + 1}: {sum(range(a, b + 2, 2))}")

File: c++/10783.cpp
#include <iostream>

using namespace std;

int main() {
    int T;
    int a, b;
    int sum;
    cin >> T;
    for(int i = 1; i <= T; i++) {
        cin >> a;
        if (a % 2 == 0) {
            a++;
        }
        cin >> b;
        sum = 0;
        for(int j = a; j <= b; j += 2) {
            sum += j;
        }
        cout << "Case " << i << ": " << sum << endl;
    }
}